### ISSUE TITLE
Add a slider for the background glow

### ISSUE THREAD

**Issue by sdfjsfjaei-hans** [psifidotos/latte-indicator-unity#4]
I think the dock might look even better with a little less (or possibly no) glow when using flat icons / themes, so a slider would be very useful. Basically the slider would determine the alpha value of the glow. (0 = no glow, 0.5 = half glow, 1 = full glow, etc.)

Thanks again for the awesome indicator! :+1:

**Comment by psifidotos** [comment]
has already implemented in master version... with values: 0% - 100%

![image](https://user-images.githubusercontent.com/2216474/55354407-548c7c80-54ce-11e9-966e-ee51ba555ff7.png)

**Comment by sdfjsfjaei-hans** [comment]
Awesome, thanks!

**Comment by sdfjsfjaei-hans** [comment]
@psifidotos Hey just a question: When will it come to the KDE store and does it update automatically?

**Comment by psifidotos** [comment]
> @psifidotos Hey just a question: When will it come to the KDE store and does it update automatically?

when I have time :) you will be able it exactly like what you did to download it from kde store, now the button in there instead of Download it will say Update

**Comment by sdfjsfjaei-hans** [comment]
Okay, just saw the plugin was updated a week ago.

So I have updated the plugin (and just checked that there really is no update anymore), but weirdly enough, I can't find the glow option.

This is what my settings look like (can't scroll down any further):

![Untitled](https://user-images.githubusercontent.com/49168569/58352169-162a9280-7e6b-11e9-83d5-fdd2c97f3a0f.png)


I don't know if it matters, but Type is set to Dock (instead of Panel).

I'm using the most recent master version of Latte dock.

**Comment by psifidotos** [comment]
this is the current version... Have you updated the Unity Indicator for inside Latte and restart Latte afterwards?

![image](https://user-images.githubusercontent.com/2216474/58352391-10857a80-7e74-11e9-8252-87b6b7e42b7b.png)

**Comment by sdfjsfjaei-hans** [comment]
Weird.. I did restart it. I have now done it again from the command line with `--replace` and now I can see the new options. Thanks! 

Two things I noticed when the background glow is at 0%: 

1. I was wrong about it looking better with no glow when using flat icons (I really like it at 75%)
2. The background seems a bit too dark / low contrast

![unity-latte](https://user-images.githubusercontent.com/49168569/58357174-19794a80-7e7a-11e9-9ade-5c58608b218e.png)

Maybe it would look better if the background color + glow were combined: (pseudo code)

```java
actualGlowColor = alpha * glowColor

MAX_GLOW_MIX_RATIO = 0.33
backgroundColor = (1 - alpha) * MAX_GLOW_MIX_RATIO * glowColor + calcBgColorFor(appIcon) 
```

where `alpha` = Background glow percentage currently in the Unity indicator options (1 = 100%)

so the lower you'd set the the glow %, the less visible the glow would be, but the more the glow color would be mixed with the normal background color (capped here at 33%).  

But maybe I missed something and this doesn't make any sense.

Either way, it's pretty good! :+1:
